web-client/src/app/views/wallet/wallet.page.html

```diff
@@ -1,7 +1,9 @@
 <ion-header class="ion-no-border">
   <app-header>
     <ion-buttons slot="end">
-      <ion-button (click)="onRefresh()">Refresh</ion-button>
+      <ion-button (click)="onRefresh()">
```

Comment:
When a button doesn't have text, it's always a good idea to add `icon-only` to its [slot](https://ionicframework.com/docs/api/button#slots)